alembic/versions/b584b9b697c8_add_models_for_city_and_temperature.py

```diff
@@ -0,0 +1,48 @@
+"""Add models for City and Temperature
+
+Revision ID: b584b9b697c8
+Revises: 
+Create Date: 2024-12-15 16:29:28.604962
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = 'b584b9b697c8'
+down_revision: Union[str, None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('city',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(length=64), nullable=False),
+    sa.Column('additional_info', sa.String(length=255), nullable=False),
```

Comment:
The `additional_info` column is set as `nullable=False`. Ensure that this column will always have data for each entry, or consider allowing it to be nullable if some entries might not have additional information.